layers/+distributions/spacemacs-base/keybindings.el

```diff
@@ -531,16 +531,17 @@
 (spacemacs|define-transient-state window-manipulation
   :title "Window Manipulation Transient State"
   :doc (concat "
- Select^^^^              Move^^^^              Split^^                Resize^^                     Other^^
- ──────^^^^───────────── ────^^^^───────────── ─────^^─────────────── ──────^^──────────────────── ─────^^──────────────────────────────
- [_j_/_k_] down/up       [_J_/_K_] down/up     [_s_] vertical         [_[_] shrink horizontally    [_q_] quit
- [_h_/_l_] left/right    [_H_/_L_] left/right  [_S_] vert & follow    [_]_] enlarge horizontally   [_u_] restore prev layout
- [_0_-_9_] window N      [_r_]^^   rotate fwd  [_v_] horizontal       [_{_] shrink vertically      [_U_] restore next layout
- [_w_]^^   other window  [_R_]^^   rotate bwd  [_V_] horiz & follow   [_}_] enlarge vertically     [_d_] close current
- [_o_]^^   other frame   ^^^^                  ^^                     ^^                           [_D_] close other"
+ Select^^^^               Move^^^^              Split^^               Resize^^             Other^^
+ ──────^^^^─────────────  ────^^^^────────────  ─────^^─────────────  ──────^^───────────  ─────^^──────────────────
+ [_j_/_k_]  down/up       [_J_/_K_] down/up     [_s_] vertical        [_[_] shrink horiz   [_u_] restore prev layout
+ [_h_/_l_]  left/right    [_H_/_L_] left/right  [_S_] verti & follow  [_]_] enlarge horiz  [_U_] restore next layout
+ [_0_.._9_] window 0..9   [_r_]^^   rotate fwd  [_v_] horizontal      [_{_] shrink verti   [_d_] close current
```

Comment:
No problem with the response time.

I agree that `..` works best for sequences, since `,` (comma) separates different key bindings and `-` (dash) is used between keys in key chords.

Now that i re-read my previous notes section comment, where the last line said: 
>In the Layouts and Workspaces Transient States they are written starting from `0` up to `9`: `[0,9]` and `[C-0,C-9]`. That seems more logical.

I see that it might have sounded like i meant that the `,` (comma) would be more logical as a sequence. But that was not my intention, i should have written the documentation suggestion as: 
`[0..9]` and `[C-0..C-9]` 
instead of them current being written like this: 
`[1..9, 0]` and `[C-1..C-9, C-0]`.

Thanks for also updating the buffer, layout and workspace transient states 👍